packages/orbit-components/src/Popover/components/ContentWrapperMobile.js

```diff
@@ -25,23 +21,17 @@ const mobileTop = theme => theme.orbit.spaceXLarge;
 const popoverPadding = theme => theme.orbit.spaceMedium;
 
 const StyledContentWrapper = styled.div`
-  overflow: auto;
-  border-top-left-radius: 12px;
-  border-top-right-radius: 12px;
-  position: absolute;
-  left: 0;
-  width: 100%;
-  background-color: ${({ theme }) => theme.orbit.paletteWhite};
-  max-height: ${({ windowHeight, actionsHeight }) => `${windowHeight - actionsHeight - 32}px`};
```

Comment:
I can't find where this `windowHeight` was coming from before you started computing it inside 😄  Was it being passed from outside somehow…?